### ISSUE TITLE
[CRASH] Olive crashes if your user folder contains accents

### ISSUE THREAD

**Issue by TenTeypek** [olive-editor/olive#1838]
**Commit Hash** 
a55e142e

**Platform** 
Windows 10

**Summary**
Olive crashes on startup when my user folder (in C:\Users) name contains accented characters. 

**Steps to Reproduce**

1. Create a new user with username that contains accented characters (for example the letter á)
2. Switch to that user
3. Open Olive

<details><summary><strong>Crash Report</strong></summary><pre><code>

</code></pre></details>

**Additional Information**
My username contains the letters á and š.
The crash report screen shows up, but the report itself is blank and sending report fails.

**Comment by AGI-THB** [comment]
It's known.

#1794

**Comment by itsmattkc** [comment]
Yes this has been a longstanding OpenColorIO bug that I wrote a fix for a long time ago. It got merged into OCIO's upstream in October 2021, but wasn't featured in a stable release of theirs until December, and still hasn't made its way into vcpkg (which is used for installing dependencies on our Windows built system). I was hoping it would happen in a more timely manner, but I might have to file the PR with vcpkg myself at this rate.
